content/posts/contributing.md

```diff
@@ -0,0 +1,235 @@
+
+``` bash
+$ git clone git@github.com:crashoverride/yt-blog.git
+```
+
+```
+Cloning into 'yt-blog'...
+remote: Enumerating objects: 608, done.
+remote: Counting objects: 100% (608/608), done.
+remote: Compressing objects: 100% (163/163), done.
+remote: Total 1535 (delta 270), reused 541 (delta 263), pack-reused 927
+Receiving objects: 100% (1535/1535), 45.42 MiB | 4.10 MiB/s, done.
+Resolving deltas: 100% (676/676), done.
+```
+
+This project uses a hugo theme called `dream`. For information about this theme
+specifically, check out the [quickstart
+page](https://g1eny0ung.site/hugo-theme-dream/#/quick-start?id=add-quotabout-mequot). 
+Because we aren't doing any customization of the theme, we are using a
+reference to it as a submodule. This isn't cloned with a simple `git clone`, so
+we need to get that too. The following command will initialize the submodule
+and clone it in the correct location of this repository. 
+
+```bash
+$ git submodule update --init
+```
+
+```
+Submodule 'themes/dream' (https://github.com/g1eny0ung/hugo-theme-dream.git)
+registered for path 'themes/dream'
+Cloning into '/Users/crashoverride/repos/yt-blog/themes/dream'...
+Submodule path 'themes/dream': checked out
+'eacbcc3f3625ce3bd770d55f130da915fa1f7ff0'
+```
+
+Now we're ready to preview it locally!
+
+### Build the Project Locally
+
+Now that you have the project and submodule cloned, and hugo installed, you can
+preview the website locally. Execute:
+
+``` bash
+$ hugo server -D
+```
+
+To create a local server of the page, served (by default) to localhost:1313
+
+The `-D` flag will show draft posts, so as you start writing your post it will
+show up on this locally served version of the webpage. 
+
+You should get the following output:
+```
+
+                   | EN
+-------------------+------
+  Pages            | 124
+  Paginator pages  |   7
+  Non-page files   |   0
+  Static files     |  91
+  Processed images |   0
+  Aliases          |  27
+  Sitemaps         |   1
+  Cleaned          |   0
+
+Built in 281 ms
+Watching for changes in /Users/crashoverride/repos/yt-blog/{content,static,themes}
+Watching for config changes in /Users/crashoverride/yt-blog/config.toml
+Environment: "development"
+Serving pages from memory
+Running in Fast Render Mode. For full rebuilds on change: hugo server --disableFastRender
+Web Server is available at http://localhost:1313/yt-blog/ (bind address 127.0.0.1)
+Press Ctrl+C to stop
+```
+
+As you modify your post and save it, the site will be rebuilt automatically.
+Output in this process looks like the following:
+
+```
+Change detected, rebuilding site.
+2021-01-28 16:08:05.778 -0600
+Source changed "/Users/crashoverride/repos/yt-blog/content/posts/mycoolpost.md": WRITE
+Total in 32 ms
+```
+
+### Create a New Post
+
+To create a new post with the header information that will be used to attribute
+your authorship and tag the post, execute:
+
+``` bash
+$ hugo new posts/mycoolpost.md
```

Comment:
related to the `hugo server -D` instructions above: `hugo new *` adds a `draft: true` attribute to the new file (or at least it did for me?). `hugo server -D` will show it locally, but does that line need to be deleted before submitting a PR?

Reply:
Great question! I will clarify this.

Reply:
Ok @chrishavlin I added some clarifying parts to the two sections you pointed out (both to explain -D and to talk about the draft line). Does this clarify enough?